packages/iris-grid/src/formatters/DateTimeColumnFormatter.test.ts

```diff
@@ -116,7 +121,8 @@ describe('calls to iris format and time zone functions', () => {
     });
     const timestamp = Date.now();
     formatter.format(timestamp);
-    formatter.format(timestamp, '');
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    formatter.format(timestamp, '' as any);
```

Comment:
Looks like this test was written incorrectly before, passing an unexpected param type in. Instead, test what the test is asking for, an undefined format or empty format string, eg:
```suggestion
    formatter.format(timestamp, { formatString: '', label: '', type: 'type-context-custom' });
```

Reply:
Good call